TechItEasy/src/main/java/nl/novi/TechItEasy/Controllers/TelevisionsController.java

```diff
@@ -0,0 +1,69 @@
+package nl.novi.TechItEasy.Controllers;
+
+import nl.novi.TechItEasy.Exceptions.IndexOutOfBoundsException;
+import nl.novi.TechItEasy.Exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
+public class TelevisionsController {
+
+    // ik had hier eerst private final List<String> televisionDataBase = Arrays.asList("Philips 43PUS6504/12", "Nikkei NH3216SMART", "Samsung QE55Q60T", "Hitachi 43HAK6152", "Philips 55PUS7805", "Brandt B2450HD"); staan, maar dat werkte niet in de post en delete methode omdat ie er kennelijk dan geen array van maakt maar een list waarop ik de add en remove methode niet mocht uitvoeren.
+    private final ArrayList<String> televisionDataBase = new ArrayList<>();
+
+    public TelevisionsController() {
+        televisionDataBase.add("Philips 43PUS6504");
+        televisionDataBase.add("Nikkei NH3216SMART");
+        televisionDataBase.add("Samsung QE55Q60T");
+        televisionDataBase.add("Hitachi 43HAK6152");
+        televisionDataBase.add("Philips 55PUS7805");
+        televisionDataBase.add("Brandt B2450HD");
+    }
```

Comment:
Deze logic wil je niet in je constructor zie comment hierboven over hoe je dit beter aan kunt pakken